policyengine_it/parameters/gov/ade/credits/dependent_spouse/phase_out.yaml

```diff
@@ -0,0 +1,21 @@
+description: Phase-out rate of the dependent spouse tax credit
+metadata:
+  threshold_unit: currency-EUR
+  rate_unit: /1
+  period: year
+  type: single_amount
```

Comment:
If we convert this file to a marginal_rate we can avoid using anything but a .calc function